bigchaindb/backend/mongodb/query.py

```diff
@@ -153,13 +153,17 @@ def get_spent(conn, transaction_id, output):
     cursor = conn.run(
         conn.collection('bigchain').aggregate([
             {'$match': {
-                'block.transactions.inputs.fulfills.txid': transaction_id,
-                'block.transactions.inputs.fulfills.output': output
+                'block.transactions.inputs.fulfills': {
```

Comment:
I guess this is ok but querying on embedded documents is tricky.
From the mongodb documentation:
> Equality matches on the whole embedded document require an exact match of the specified <value> document, __including the field order__.